lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -103,14 +110,18 @@ def workspace_did_change_watched_files(changes)
           end
         end.compact
 
-        return if constants.empty? && !has_route_change
+        return if constants.empty? && !has_route_change && !has_fixtures_change
```

Comment:
If we continue adding other things to watch then this will get unwieldy, but I think it's ok for now.

Reply:
Why can't we fold them both into a single boolean?

Reply:
Ah, I see that we run different DSLs based on the flags.